traitsui/testing/tester/ui_wrapper.py

```diff
@@ -80,6 +81,7 @@ def locate(self, location):
         return UIWrapper(
             target=self._get_next_target(location),
             registries=self._registries,
+            delay=self.delay,
```

Comment:
I think we need a test to check that the `delay` parameter is persisted throughout after calling `UITester.find_by_name`.